src/components/boardComponents/ItemFormFactory.jsx

```diff
@@ -104,13 +104,13 @@ const ItemFormFactory = () => {
                     step={5}
                     included={false}
                     marks={{
+                      "-180": -180,
                       "-90": -90,
                       "-45": -45,
-                      "-30": -30,
```

Comment:
The 30° value is convenient for hexagons ;-)

Reply:
Got it, I need to change the font size then.